examples/basic.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
Are you wanting to build out examples in this new examples directory and build the `gh-pages` off of that.